data/json/items/gun/shot.json

```diff
@@ -89,7 +88,10 @@
         "volume": 1,
         "price": 5000
     },
-    "burst": 2,
+    "modes": [
+      [ "DEFAULT", "single", 1 ],
+      [ "AUTO", "double", 2 ]
```

Comment:
Can I change the "AUTO" mode name without breaking the game? I thought it had to be named AUTO or the game would glitch.